client/coral-framework/store/actions/items.js

```diff
@@ -241,9 +241,45 @@ export function postAction (item_id, action_type, user_id, item_type) {
           return response.ok ? response.json()
           : Promise.reject(`${response.status} ${response.statusText}`);
         }
-      )
-      .then((json)=>{
-        return json;
-      });
+      );
+  };
+}
+
+/*
+* DeleteAction
+* Deletes an action to an item
+*
+* @params
+*   id - the id of the item on which the action is taking place
+*   action - the name of the action
+*   user - the user performing the action
+*   host - the coral host
+*
+* @returns
+*   A promise resolving to null or an error
+*
+*/
+
+export function deleteAction (item_id, action_type, user_id, item_type) {
+  return () => {
+    const action = {
+      action_type,
+      user_id
+    };
+    const options = {
```

Comment:
I would recommend pulling these options hashes out to an init function like [here](https://github.com/coralproject/talk/blob/master/client/coral-admin/src/actions/settings.js#L13) if only because we're going to be adding auth to all this stuff, and it make everything easier.